server_addon/substancepainter/client/ayon_substancepainter/deploy/plugins/ayon_plugin.py

```diff
@@ -1,34 +1,34 @@
 
 
-def cleanup_openpype_qt_widgets():
+def cleanup_ayon_qt_widgets():
     """
         Workaround for Substance failing to shut down correctly
         when a Qt window was still open at the time of shutting down.
 
         This seems to work sometimes, but not all the time.
 
     """
-    # TODO: Create a more reliable method to close down all OpenPype Qt widgets
+    # TODO: Create a more reliable method to close down all AYON Qt widgets
     from PySide2 import QtWidgets
     import substance_painter.ui
 
-    # Kill OpenPype Qt widgets
-    print("Killing OpenPype Qt widgets..")
+    # Kill AYON Qt widgets
+    print("Killing AYON Qt widgets..")
     for widget in QtWidgets.QApplication.topLevelWidgets():
-        if widget.__module__.startswith("openpype."):
+        if widget.__module__.startswith("ayon."):
```

Comment:
Wouldn't this technically be either `ayon_core.` or even `ayon_substancepainter.` if the Qt widgets are in substance painter integration?